homeassistant/components/environment_canada/__init__.py

```diff
@@ -87,6 +89,17 @@ async def async_unload_entry(hass: HomeAssistant, config_entry: ConfigEntry) ->
     return unload_ok
 
 
+def device_info(config_entry: ConfigEntry) -> DeviceInfo:
+    """Build and return the device info for EC."""
+    return DeviceInfo(
+        entry_type=DeviceEntryType.SERVICE,
+        identifiers={(DOMAIN, config_entry.entry_id)},
+        manufacturer="Environment Canada / Environnement Canada",
```

Comment:
Lame change maybe, codespell linter triggers on it and I don't see a simpler way around it right now.

```suggestion
        manufacturer="Environment Canada",
```

Reply:
I was trying to respect the official name of the government department.